pkgs/development/python-modules/swagger-ui-bundle/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, jinja2, flake8 }:
+
+buildPythonPackage rec {
+  pname = "swagger-ui-bundle";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
```

Comment:
Is this distributed on `PyPi`?
If so we should use `fetchPypi`.

That's preferred usually, and since there's no tests anywhere that shouldn't be problem with them not having them..